fetch-node.js

```diff
@@ -1,18 +1,17 @@
 'use strict';
 
 var fetch = require('node-fetch');
+var URL = require('url').URL;
```

Comment:
This line is redundant now.

Reply:
What should I do?

If I rewrite it I use the scope. Then change var to const.
```diff
-var fetch = require('node-fetch');
+const fetch = require('node-fetch');
-var URL = require('url').URL;
+const { URL } = require('url');
```

But the scope throws an error with eslint. So specify 6 for ecmaVersion.
```diff
 // .eslintrc.json
     },
     "globals": {
         "Promise": true,
         "XMLHttpRequest": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 6
     }
 }
```

If this is no problem, I will rewrite it right now

Reply:
This particular line was introduced by this pull request, but `URL` is no longer used, so I'd like it to be removed again.

Reply:
Ah, I see.